### PR TITLE
Fix for UI Breaking Issue on Status Page for Smaller Devices and Tablets

### PR DESCRIPTION
**Pull request by aneenajohn**
### Description:
Need to enhance the responsiveness of the status page

## What has changed?
- The status page UI issues breaking on tab-sized and small laptops like 1024px and smaller mobile devices 280px(as in Galaxy Fold) and 320px(iPhone5) have been fixed

### Dev Tested:
- [✔] Yes

### Images/video of the change:
[Click here for the demo video for responsiveness of status site](https://drive.google.com/file/d/1b1N-aVKV6yBKrAzhAmWk5B38jq9FOpUg/view?usp=sharing)

### Follow-up Issues (if any)
https://github.com/Real-Dev-Squad/website-status/issues/866

### CODE REVIEW

File: src/components/tasks/card/card.module.scss [hunk 1]
```diff
@@ -364,6 +380,14 @@ $progressTextAnimation: #9babb8;
     flex-direction: column;
     justify-content: center;
     align-items: center;
+    @media (max-width: 28rem) {
+        padding: 1rem;
+    }
+
+    // 368px
+    // @media (max-width: 23rem) {
+    //     padding: 0.5rem ;
```
Comment (Maheima) [inline src/components/tasks/card/card.module.scss:389]: Please remove this piece of commented code

File: src/components/tasks/card/card.module.scss [hunk 2]
```diff
@@ -384,6 +408,15 @@ $progressTextAnimation: #9babb8;
     width: 100%;
     display: flex;
     gap: 5px;
+
+    @media (max-width: 28rem) {
```
Comment (Maheima) [inline src/components/tasks/card/card.module.scss:412]: Either use "rem" values everywhere for media queries or just "px" values.

**Review by Maheima — DISMISSED** [review]

**Comment by aneenajohn** [comment]
> Can you please add working video

Please click on the demo video link above

File: src/components/tasks/card/card.module.scss (hunk 1 above)
Comment (aneenajohn) [reply src/components/tasks/card/card.module.scss]: Removed comments

File: src/components/tasks/card/card.module.scss (hunk 2 above)
Comment (aneenajohn) [reply src/components/tasks/card/card.module.scss:407]: Have used "rem" in those modules as in card scss module wherein "rem" is used in all other places over this module and "px" in those modules like task & taskSearch wherein "px" is used on media query for all other places. Please let me know if we have to update this

**Review by iamitprakash — DISMISSED** [review]

File: src/components/tasks/card/card.module.scss (hunk 2 above)
Comment (Maheima) [reply src/components/tasks/card/card.module.scss:406]: What do you think we should go ahead with? @bhtibrewal @shreya-mishra @harshith-venkatesh @iamitprakash

Comment (aneenajohn) [reply src/components/tasks/card/card.module.scss:406]: @Maheima @iamitprakash @bhtibrewal @shreya-mishra @harshith-venkatesh @sahsisunny Please provide suggestions on this

**Review by sahsisunny — APPROVED** [review]

File: src/components/tasks/card/card.module.scss (hunk 2 above)
Comment (aneenajohn) [reply src/components/tasks/card/card.module.scss]: @Maheima Updated the updated media queries in rem to respective px. Please review

**Review by iamitprakash — APPROVED** [review]

**Pull request merged**